charts/kubernetes-external-secrets/README.md

```diff
@@ -0,0 +1,123 @@
+# 💂 Kubernetes External Secrets
+
+[Kubernetes External Secrets](https://github.com/godaddy/kubernetes-external-secrets) allows you to use external secret management systems (*e.g.*, [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)) to securely add secrets in Kubernetes. Read more about the design and motivation for Kubernetes External Secrets on the [GoDaddy Engineering Blog](https://godaddy.github.io/2019/04/16/kubernetes-external-secrets/).
+
+## TL;DR;
+
+```bash
+$ helm install stable/kubernetes-external-secrets
+```
+
+## Prerequisites
+
+* Kubernetes 1.7+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/kubernetes-external-secrets
+```
+
+> **Tip:** A namespace can be specified by the `Helm` option '`--namespace kube-external-secrets`'
+
+## Uninstalling the Chart
+
+To uninstall/delete the `kubernetes-external-secrets` deployment:
+
+```bash
+helm delete kubernetes-external-secrets
```

Comment:
Kind of a nit, but if you follow the instructions above, I think you need to run `helm delete my-release`

Reply:
ah, yeah, updated